integration-tests/src/test/java/oracle/kubernetes/operator/utils/LoadBalancer.java

```diff
@@ -150,10 +150,26 @@ private void upgradeTraefikNamespace() throws Exception {
     LoggerHelper.getLocal().log(Level.INFO, " upgradeTraefikNamespace() Running " + cmd.toString());
     ExecResult result = ExecCommand.exec(cmd.toString());
     if (result.exitValue() != 0) {
+      TestUtils.checkHelmChart("traefik-operator","traefik");
       reportHelmInstallFailure(cmd.toString(), result);
     }
     String outputStr = result.stdout().trim();
     LoggerHelper.getLocal().log(Level.INFO, "Command returned " + outputStr);
+    int i = 0;
+    //wait until pod is restarted after upgrade and check the status
+    String traefikPod = TestUtils.getPodName(" -l app=traefik ", "traefik");
+    cmd = new StringBuffer();
+    cmd.append("kubectl get pod -n traefik");
+    while (i < maxIterationsPod) {
+      if (TestUtils.checkPodContains(cmd.toString(), "Terminating", traefikPod)) {
```

Comment:
traefik (voyager) pod at same cases terminates and restarted in others it does not, after adding check for creation timestamp some tests failed because timestamp check ( so I removed it back). Added status of release check

Reply:
I checked with Ryan, he said traefik pod should not be getting restarted after upgrade. You can remove the check you have here for pod restart.

Reply:
the traffic pod in some cases get restarted, keeping the check